django_readers/pairs.py

```diff
@@ -1,9 +1,14 @@
 from django.db.models import Count
-from django_readers import projectors, qs
+from django_readers import producers, projectors, qs
+
+
+def wrap_producer(name, pair):
```

Comment:
Maybe call this `project` instead of `wrap_producer`?

Reply:
I think that might be confusing, it feels to me like it implies that the thing being "projected" is the pair, which it isn't.

Other names I considered for this function: `wrap_producer_with_projector`, `convert_producer_to_projector`, `producer_to_projector`, `make_projector`. What do you think of those?

Reply:
The reason I went with `wrap_producer` is that the same function name kinda works whether it's `projectors.wrap_producer` or `pairs.wrap_producer`. But that might be true of some of the others too.

Reply:
Gone with `producer_to_projector`